### PR TITLE
docs: cleaner README.md

### PR DESCRIPTION
**Pull request by davlgd**

### CODE REVIEW

File: docs/log-drains.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+
+### Logs Drains
```
Comment (aurrelhebert) [inline docs/log-drains.md:2]: Do we agree that this page is no longer linked on the main page? If I remember correctly the only way to activate the logs drain is threw the CLI, no?

**Review by aurrelhebert — COMMENTED** [review]
LGTM  only a single comment :)

File: docs/log-drains.md (hunk 1 above)
Comment (davlgd) [reply docs/log-drains.md:2]: As far as I know, this doc should be rewrote here and in Clever Doc. I'll link it back when I'll be sure the content is OK. The goal was first to get a cleaner README and the iterate on this quickly

**Review by aurrelhebert — APPROVED** [review]

File: docs/setup-systems.md [hunk 2]
```diff
@@ -0,0 +1,180 @@
+
```
Comment (hsablonniere) [inline docs/setup-systems.md:1]: * Can we add a level 1 title?

File: docs/setup-systems.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+### GNU/Linux
```
Comment (hsablonniere) [inline docs/setup-systems.md:2]: Can we lower all titles from one level?

File: README.md [hunk 4]
```diff
@@ -1,14 +1,10 @@
-# clever-tools
+# Clever Tools
 
-Command Line Interface for Clever Cloud.
+Deploy on Clever Cloud and control you applications, addons, services from command line.
```
Comment (hsablonniere) [inline README.md:3]: The official spelling is `add-on`. Don't ask me.

File: docs/log-drains.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+
```
Comment (hsablonniere) [inline docs/log-drains.md:1]: Can we add a level 1 title and lower the other titles of 1 level?

**Review by hsablonniere — CHANGES_REQUESTED** [review]

**Review by miton18 — APPROVED** [review]

**Pull request merged**